packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -532,7 +524,13 @@ contract Deploy is Deployer {
         // are always proxies.
         Types.ContractSet memory contracts = _proxiesUnstrict();
         contracts.L2OutputOracle = address(oracle);
-        ChainAssertions.checkL2OutputOracle(contracts, cfg, 0, 0);
+        ChainAssertions.checkL2OutputOracle({
+            _contracts: contracts,
+            _cfg: cfg,
+            _l2OutputOracleStartingBlockNumber: 0,
```

Comment:
Aren't these values present in the config? Why the need to break out additional params to this function?